sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -15,9 +19,72 @@
 from sunpy.time import parse_time, is_time_equal
 
 
-@pytest.fixture
-def LCClient():
-    return goes.XRSClient()
+LCClient = goes.XRSClient()
+
+
+def mock_query_object(start_date, end_date):
+    """
+    Creation of a QueryResponse object, and prefill some
+    downloaded data from goes.XRSClient().fetch(Time('20 ..)
+    """
+    # Create a mock QueryResponse object
+    map_ = {
+        'TimeRange' : TimeRange(parse_time(start_date), parse_time(end_date)),
+        'Time_start': parse_time(start_date),
+        'Time_end':  parse_time(end_date),
+        'source': 'nasa',
+        'instrument': 'goes',
+        'physobs': 'irradiance',
+        'provider': 'sdac'
+    }
+
+    st_datetime = datetime.strptime(start_date, '%Y/%m/%d')
+    ed_datetime = datetime.strptime(end_date, '%Y/%m/%d') + timedelta(days=1)
+    time_range = TimeRange(st_datetime, ed_datetime).split(int((ed_datetime-st_datetime).days))
```

Comment:
Isn't this the same as
```py
tr = map_['TimeRange']
time_range = tr.split(int((tr.days + 1).value))
```

Reply:
The `TimeRange` object is getting split into parts, which is not what I intend. So I am dropping this suggestion